test/cookbooks/osl-nginx-test/recipes/etherpad.rb

```diff
@@ -1,12 +1,3 @@
-# Resolves a weird issue in the etherpad suite where the nginx service will be
-# successfully started by Chef but only listen on 80 and not 8090/443. If you
-# go into the VM and manually try to (re)start the service, it will fail
-# because SELinux blocks this port. Fixing this SELinux issue also resolves
-# nginx not listening to 443 for some reason.
-execute 'semanage port -a -t http_port_t  -p tcp 8090' do
-  not_if 'semanage port -l | grep http_port_t | grep 8090'
-end
-
```

Comment:
Why is this being removed?

Reply:
It breaks on centos 8, also I didnt notice a difference in listening ports with this in/out on 7 so I am not sure what its purpose is?